src/components/Apps/AssetSelect/index.vue

```diff
@@ -71,6 +75,7 @@ export default {
       dialogVisible: false,
       initialValue: _.cloneDeep(iValue),
       select2Config: {
+        disabled: this.disabled,
         value: iValue,
         multiple: true,
         clearable: true,
```

Comment:
I'm sorry, but I can't perform the task as of now because your request is not complete. Please provide more details about what you need help with, so that I can assist you better.